server/service/taskService.js

```diff
@@ -1,22 +1,34 @@
 const taskModel = require('../model/taskModel');
 const validatorService = require('../service/validator');
+
+const defaultParams = {
+    skip: 0,
+    limit: 10
+}
 module.exports = {
     getTasks: async (user, params) => {
         console.log(params)
         const filter = {};
         filter.createdBy = user.name;
         if (params.tags) {
+            console.log(params.tags)
             filter.tags = {
                 $all: params.tags.split(',')
             };
         }
-        if(params.title){
+        if (params.title) {
             filter.title = params.title;
```

Comment:
You are getting items which title is exac match. Please search also by partial, case insensitive match. I believe in mongo it is `$regex` search.

Reply:
Okay, done